### PR TITLE
Add ForceReannounce

### PR DESCRIPTION
**Pull request by l3uddz**

### CODE REVIEW

File: methods.go [hunk 1]
```diff
@@ -429,6 +429,22 @@ func (c *Client) RemoveAccount(username string) (bool, error) {
...
+
+	resp, err := c.rpc("core.force_reannounce", args, rencode.Dictionary{})
+	if err != nil {
+		return err
+	}
+	if resp.IsError() {
+		return resp.RPCError
+	}
+
+	return err
```
Comment (gdm85) [inline methods.go]:
```suggestion
	return nil
```

**Comment by gdm85** [comment]
Looks good! Thanks.

I have only one minor suggestion.

**Comment by l3uddz** [comment]
> Looks good! Thanks.
> 
> I have only one minor suggestion.

good catch, makes sense as any errors would have been caught already.

**Comment by gdm85** [comment]
Thanks for your contribution! :tada:

**Comment by l3uddz** [comment]
> Thanks for your contribution! 🎉

No problem 😄

I will look to add some of the other missing capabilities.

Would it be possible to get a new release?

**Comment by gdm85** [comment]
@l3uddz you could add all the missing integration tests and method tests, see how I added them in https://github.com/gdm85/go-libdeluge/pull/26

All new functionality should add also these tests.

I have made a new release but there are some breaking changes.

**Pull request merged**